### ISSUE TITLE
App crashed

### ISSUE THREAD

**Issue by navneetccna** [mihir-harbinger/commonDemo#1]
I am using andorid and when i start app it just crashed.

**Comment by mihir0x69** [comment]
@navneetccna Hey I'm sorry, I'll have set this up on my machine. It's been ages since I touched this one. I'll get back to you as soon as possible.

**Comment by navneetccna** [comment]
Ok, I am waiting for the solution

Thanks

**Comment by mihir0x69** [comment]
@navneetccna If you're just learning react-native, you can [refer to this one](https://github.com/KarandikarMihir/IOT) as well

**Comment by navneetccna** [comment]
Yes, i am learning react-native. Please share more projects if you can.

Thanks

**Comment by mihir0x69** [comment]
Sorry that's all I've got. I moved to web from native.
